app/models/notification.rb

```diff
@@ -17,13 +17,22 @@ class Notification < ApplicationRecord
   scope :read,    -> { where.not(read_at: nil) }
   scope :unread,  -> { where(read_at: nil) }
 
+  # All unread notifications within a given span of time
+  scope :since, -> (time_ago = 1.day) {
```

Comment:
are we using this a lot in non spec code? I see it just once. 
I like the scope but a private method could be enough.

Reply:
I'm hesitant with moving this to a private method just because we lose the advantage of scopes always returning an ActiveRecord::Relation. In a scope, we can be sure that we can always chain the result.

Reply:
It can still be a private AR scope if that works. Move the line under `private` and it applies the function like any other.

Reply:
^ doesnt' seem to work. Adding this line does though: `private_class_method :since`. But I'm not convinced we should make it private in the first place?

Reply:
> doesnt' seem to work

huh. Today I learned.

I feel it's fine to leave it as is then.